test/e2e/setup.go

```diff
@@ -423,3 +438,17 @@ var _ = AfterSuite(func() {
 		log.Printf(err.Error())
 	}
 })
+
+var _ = JustAfterEach(func() {
+	gsc := CurrentSpecReport()
+	metricName := strings.Replace(gsc.ContainerHierarchyTexts[0], " ", ".", -1)
+
+	log.WithFields(logrus.Fields{
+		e2emetrics.LogEntryIsE2EEmittableMetric: true,
+		e2emetrics.LogEntryMetricName:           metricName,
+		e2emetrics.LogEntryMetricStatus:         !gsc.Failed(),
```

Comment:
That's a good point. Here the bool doesn't represent a status. (see my other suggestion)

Reply:
@facchettos - I updated the names (variables, log entry fields, metrics dimensions) to be more representative of state/bool. Is this along the lines of what you were looking for?